source/application/AMPIC887DriverServer/GCS2Commands/AMGCS2GetMinCommandablePositionCommand.cpp

```diff
@@ -51,56 +38,29 @@ bool AMGCS2GetMinCommandablePositionCommand::validateArguments()
 
 bool AMGCS2GetMinCommandablePositionCommand::runImplementation()
```

Comment:
Honestly, it almost looks like if you could somehow pass the handler for each command you wouldn't even need to have these subclassed/implemented runImplementations().